VerloopSDK/VLViewController.swift

```diff
@@ -75,8 +84,32 @@ class VLViewController: UIViewController, WKUIDelegate {
             noNetworkAlert.addAction(okAction)
             self.present(noNetworkAlert, animated: true, completion: nil)
             loader?.removeFromSuperview()
+            return
         }
-        self.updateWebViewConstraints()
+        loader = UIActivityIndicatorView(frame: CGRect(x: view.center.x, y: view.center.y, width: 30, height: 30))
+        loader?.tintColor = .black
+        if #available(iOS 13.0, *) {
+            loader?.style = .large
+        }
+        loader?.center = self.view.center
+        loader?.startAnimating()
+        view.addSubview(loader!)
```

Comment:
this will crash if loader is nil

Reply:
when you are creating references that can be optional but when we are adding a view as subview that cannot be optional and we have added this instance just above so there is no possibility of this being nil.